libraries/botbuilder/src/botFrameworkAdapter.ts

```diff
@@ -544,6 +550,15 @@ export class BotFrameworkAdapter extends BotAdapter implements ConnectorClientBu
      */
     public async getUserToken(context: TurnContext, connectionName: string, magicCode?: string): Promise<TokenResponse>;
     public async getUserToken(context: TurnContext, connectionName: string, magicCode?: string, oAuthAppCredentials?: CoreAppCredentials): Promise<TokenResponse>;
+    /**
+     * Asynchronously attempts to retrieve the token for a user that's in a login flow.
+     * @param context The context object for the turn.
+     * @param connectionName The name of the auth connection to use.
+     * @param magicCode Optional. The validation code the user entered.
+     * @param oAuthAppCredentials Optional. AppCredentials for OAuth.
+     * 
+     * @returns A [TokenResponse](xref:botframework-schema.TokenResponse) object that contains the user token.
```

Comment:
Wen clicking on the xref in vs code I get an error. Is this the proper way to do this?
![image](https://user-images.githubusercontent.com/38112957/94469810-19d7d300-019d-11eb-8ed0-88c9027db81e.png)

Also, I think we are not using a space between the params and returns.

Reply:
In this case, I'd keep it like this, as it's a method overload and it has the almost exact documentation that the methods above.